kde-apps/umbrello/umbrello-24.12.3.recipe

```diff
@@ -17,6 +17,8 @@ SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
 	umbrello$secondaryArchSuffix = $portVersion
+	cmd:po2xmi5$secondaryArchSuffix = $portVersion
+	cmd:xmi2pot5$secondaryArchSuffix = $portVersion
 	app:Umbrello$secondaryArchSuffix = $portVersion
```

Comment:
```suggestion
	app:Umbrello = $portVersion
```

Also, this should probably use $commandBinDir and drop the suffix from the commands as well.